### ISSUE TITLE
[java] FieldNamingConventions: false positive in serializable class with serialVersionUID

### ISSUE THREAD

**Issue by regrog** [pmd/pmd#1329]
**Affects PMD Version:** 
6.7.0

**Rule:**
https://pmd.github.io/pmd-6.7.0/pmd_rules_java_codestyle.html#fieldnamingconventions

**Description:**
After updating PMD from version 6.6.0 to 6.7.0 all these false positive appear. 
I like all these new checks but I think that in a serializable class, `serialVersionUID` souldn't be considered.

**Code Sample demonstrating the issue:**

```
public class MyException extends RuntimeException {

	private static final long serialVersionUID = -8134636876462178354L;

	public MyException(String message, Throwable cause) {
		super(message, cause);
	}

	public MyException(String message) {
		super(message);
	}

}
```

**Running PMD through:**
Gradle 4.10
java 8

Thanks in advance

**Comment by jsotuyod** [comment]
@regrog thanks for the report. You are absolutely right about this one.
